### PR TITLE
fix: prevent undefined array key exception

### PR DESCRIPTION
**Pull request by helsner**

### CODE REVIEW

**Review by twojtylak — CHANGES_REQUESTED** [review]
Warning: ] 3 errors are fixable! Just add "--fix" to console command and rerun  
           to apply.

File: Configuration/TCA/Overrides/be_users.php [hunk 1]
```diff
@@ -1,8 +1,10 @@
 <?php
 
 use TYPO3\CMS\Core\Utility\ExtensionManagementUtility;
+
 // And add it to showitem
-if (false === \strpos($GLOBALS['SiteConfiguration']['site']['types']['0']['showitem'], '--div--;API,')) {
+if (isset($GLOBALS['SiteConfiguration']['site']['types']['0']['showitem'])
```
Comment (tmaroschik) [inline Configuration/TCA/Overrides/be_users.php:6]: I think this whole if can be dropped, as this is duplicated here https://github.com/DFAU/toujou-oauth2-server/blob/master/Configuration/SiteConfiguration/Overrides/sites.php#L19

**Review by tmaroschik — CHANGES_REQUESTED** [review]

File: Configuration/TCA/Overrides/be_users.php (hunk 1 above)
Comment (helsner) [reply Configuration/TCA/Overrides/be_users.php:6]: lmao, k will do
did not even realize, SiteConfig stuff makes no sense in the be_users TCA

Comment (helsner) [reply Configuration/TCA/Overrides/be_users.php]: updated

**Pull request merged**